docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,141 @@
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers).
+Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+There are two issues to enable ``--version`` option in ``Proxy`` and ``Shim``.
+This way the end user can know for sure which version of these components are
+installed and running.
+
+  * [https://github.com/clearcontainers/proxy/issues/69](https://github.com/clearcontainers/proxy/issues/69)
+  * [https://github.com/clearcontainers/shim/issues/32](https://github.com/clearcontainers/shim/issues/32)
+
+## Setup the environment
+
+1. Define GOPATH
+
+```bash
+$ export GOPATH=$HOME/go
+```
+
+2. Create GOPATH Directory
+
+```bash
+$ mkdir -p $GOPATH
+```
+
+3. Get the code
+
+```bash
+$ go get github.com/clearcontainers/runtime
+$ go get github.com/clearcontainers/proxy
+$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers
+```
+
+## Build and install components
+
+1. Proxy
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/proxy
+$ make
+$ sudo make install
+```
+
+2. Shim
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/shim
+$ ./autogen.sh
+$ make
+$ sudo make install
+```
+
+3. Runtime
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/runtime
+$ make
+$ sudo make install
+```
+
+4. Virtcontainers (pause binary only)
+
+This step will only install the **pause** binary included in [https://github.com/containers/virtcontainers](https://github.com/containers/virtcontainers)
+
+The ``pause`` binary is required to allow the creation of an "empty" pod.
+The pod does not contain any containers; it simply provides the environment
+to allow their creation.
+
+```bash
+$ sudo yum install -y glibc-static
+$ cd $GOPATH/src/github.com/clearcontainers/runtime/.ci/
+$ ./install_virtcontainers.sh
+```
+
+## Enable Clear Containers 3.0 for Docker
+
+1. Create a link of Clear Containers configuration file
+
+This step is needed due to issue [#206](https://github.com/clearcontainers/runtime/issues/206)
+
+
+```bash
+$ mkdir -p /etc/clear-containers
+$ ln -s /usr/local/etc/clear-containers/configuration.toml /etc/clear-containers/configuration.toml
+```
+
+Edit the file according to your needs.
+
+Refer to [https://github.com/clearcontainers/runtime#global-logging](https://github.com/clearcontainers/runtime#global-logging)
+for additional information.
+
+2. Configure Docker for Clear Containers 3.0
+
+```bash
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/local/bin/cc-runtime --default-runtime=runc
```

Comment:
We've used the `cor` name unofficially for CC 2.1, but we need a new name to refer to the CC 3.0 runtime to avoid confusion (I personally use `cc30` which isn't very "catchy", but is different ;)